keyboards/converter/usb_usb/leonardo/rules.mk

```diff
@@ -0,0 +1,2 @@
+# Processor frequency
+F_CPU = 16000000
```

Comment:
```suggestion
# This file intentionally left blank
```
Technically `F_CPU` is not needed for this or the `hasu` revision, as it's the default for 32U4.

Reply:
Thanks for the comments. I updated the PR to comment out F_CPU for this and hasu.